### ISSUE TITLE
MissingMethodException: Microsoft.VisualStudio.ProjectSystem.ProjectItemIdentity.get_LinkPath

### ISSUE THREAD

**Issue by loop-evgeny** [dotnet/project-system#3189]
After building the latest code on master and running it I tried to open a solution containing C# and F# projects in the "experimental" VS instance. An unhandled exception occurred while loading an F# project:

```
System.MissingMethodException: 'Method not found: 'System.String Microsoft.VisualStudio.ProjectSystem.ProjectItemIdentity.get_LinkPath()'.'
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Order.TreeItemOrderPropertyProvider.GetDisplayPath(UnconfiguredProject project, ProjectItemIdentity item)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Order.TreeItemOrderPropertyProvider.CreateOrderedMap(UnconfiguredProject project, IReadOnlyCollection`1 orderedItems)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Order.TreeItemOrderPropertyProvider..ctor(IReadOnlyCollection`1 orderedItems, UnconfiguredProject project)
   at Microsoft.VisualStudio.ProjectSystem.VS.Tree.Order.TreeItemOrderPropertyProviderSource.<LinkExternalInput>b__3_0(IProjectVersionedValue`1 orderedItems)
   at System.Threading.Tasks.Dataflow.TransformBlock`2.ProcessMessage(Func`2 transform, KeyValuePair`2 messageWithId)
```

**Comment by Pilchie** [comment]
@jmarolf @natidea @TIHan - I suspect this is because we have a dependency on a newer CPS than was in @loop-evgeny's VS?  Do we know if there is a public VS with a new enough CPS? If so, what is it?

**Comment by TIHan** [comment]
Yes, this is due to a newer version of COS. Updating to preview 15.6 preview 4 should make this work.

**Comment by Pilchie** [comment]
Does it work with Preview 3?  Preview 4 isn't publicly available yet...

**Comment by TIHan** [comment]
Checking versions of CPS, from my understanding, it should be in Preview 3.

**Comment by davkean** [comment]
Closing this out. This should be fixed in 15.6 Preview 4 and later. Please open a new bug if you are still running into issues.
